src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -444,11 +444,21 @@
                                                                     BorderThickness="0,0,0,0"
                                                                     ContentAlignment="Left"
                                                                     CornerRadius="0">
-                                                                    <controls:CheckBoxWithDescriptionControl
-                                                                        Margin="56,0,0,0"
-                                                                        Description="{x:Bind Path=DisplayDescription}"
-                                                                        Header="{x:Bind Path=DisplayLabel}"
-                                                                        IsChecked="{x:Bind Path=Value, Mode=TwoWay}" />
+                                                                    <StackPanel Orientation="Vertical">
+                                                                        <controls:CheckBoxWithDescriptionControl
+                                                                            Margin="56,0,0,0"
+                                                                            Description="{x:Bind Path=DisplayDescription}"
+                                                                            Header="{x:Bind Path=DisplayLabel}"
+                                                                            IsChecked="{x:Bind Path=Value, Mode=TwoWay}"
+                                                                            Visibility="{x:Bind Path=ShowCheckBox, Converter={StaticResource BoolToVisibilityConverter}}"/>
+                                                                        <ComboBox
+                                                                            Margin="56,0,0,0"
+                                                                            Description="{x:Bind Path=DisplayDescription}"
```

Comment:
Please make sure that it is possible to go without description. Current it is optional.

Reply:
Its optional. Not used